operations/products.md

```diff
@@ -47,9 +47,21 @@ Returns all products offered together with the specified services. Note this ope
             "AccountingCategoryId": "6535e19e-1077-49d9-a338-67bf4ffecb14",
             "IsActive": true,
             "Name": "Breakfast",
```

Comment:
Hi, the deprecated fields should be removed from the JSON samples - Name (singular), ExternalName (singular), etc.